examples/nidaqmx_analog_input/NIDAQmxAnalogInput.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.NIDAQmxAnalogInput_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "Measurement example that uses NI-DAQmx to measure analog input.",
+        "ni/service.collection": "NI.Examples",
+        "ni/service.tags": []
```

Comment:
Should the examples have any tags?

Reply:
They had the same question on Curtis [PR](https://dev.azure.com/ni/DevCentral/_git/ASW/pullRequest/510451#1687296746)  and the answer was nop.